common-app-api/src/main/java/org/openecomp/sdc/common/http/client/api/HttpRequestHandler.java

```diff
@@ -79,13 +79,27 @@ public static void setTestInstance(HttpRequestHandler handlerInstance) {
     }
 
     public HttpResponse<byte[]> getAsByteArray(String url, Properties headers, HttpClientConfig config) throws HttpExecuteException {
-        HttpClient client = getOrCreateClient(url, config);
-        return client.<byte[]>get(url, headers, byteResponseBuilder);
+        HttpClient client = null;
+        try {
```

Comment:
What about using try with resources ? `try(HttpClient client = getOrCreateClient(url, config)) {` ?

Reply:
Yes I thought about that too.
Unfortunately the class `HttClient` is a custom one which does not implement the `AutoCloseable` interface.

Reply:
Yes I thought about that too.
Unfortunately the class `HttClient` is a custom one which does not implement the `AutoCloseable` interface.

Reply:
Fair enough LGTM